e2e-tests/portal/cypress/integration/journeys/maker/dashboard/gef-deals/dashboard-table-data.spec.js

```diff
@@ -71,3 +74,102 @@ context('View a deal', () => {
+  ];
+
+  const mockExporters = [
+    mockExporter.exporterOne,
+    mockExporter.exporterTwo,
+    mockExporter.exporterThree,
+    mockExporter.exporterFour,
+  ];
+
+  beforeEach(() => {
+    // same or gef
+    cy.deleteGefApplications(MAKER_LOGIN);
+    cy.insertManyGefApplications(mockDeals, MAKER_LOGIN).then((insertedDeals) => {
+      deals = insertedDeals;
+      cy.insertManyGefExporters(deals, mockExporters, MAKER_LOGIN).then((insertedExporter) => { exporters = insertedExporter; });
+    });
+  });
+
+  it('multiple GEF deals appears on the dashboard', () => {
+    // login and go to dashboard
+    cy.login(MAKER_LOGIN);
+    gefDashboard.visit();
+
+    // ensures all deals are in the table
+    gefDashboard.rows().should('have.length', mockDeals.length);
+    // in order of insertion
+    cy.get('table tr').eq(1).find(`[data-cy="deal__exporter--${deals[3]._id}"]`).should('exist');
+    cy.get('table tr').eq(2).find(`[data-cy="deal__exporter--${deals[2]._id}"]`).should('exist');
+    cy.get('table tr').eq(3).find(`[data-cy="deal__exporter--${deals[1]._id}"]`).should('exist');
+    cy.get('table tr').eq(4).find(`[data-cy="deal__exporter--${deals[0]._id}"]`).should('exist');
+  });
+
+  it('sort alphabetically (ascending first)', () => {
+    cy.login(MAKER_LOGIN);
+
+    // login and go to dashboard
+    gefDashboard.visit();
+
+    // ensures the sort is set to none (default) at start
+    gefDashboard.tableHeader('exporter').invoke('attr', 'aria-sort').should('eq', 'none');
+
+    gefDashboard.tableHeader('exporter').find('button').click();
+    // ensures set to ascending
+    gefDashboard.tableHeader('exporter').invoke('attr', 'aria-sort').should('eq', 'ascending');
+
+    // check order
+    cy.get('table tr').eq(1).find(`[data-cy="deal__exporter--${deals[3]._id}"]`).should('exist');
+    cy.get('table tr').eq(2).find(`[data-cy="deal__exporter--${deals[2]._id}"]`).should('exist');
+    cy.get('table tr').eq(3).find(`[data-cy="deal__exporter--${deals[1]._id}"]`).should('exist');
+    cy.get('table tr').eq(4).find(`[data-cy="deal__exporter--${deals[0]._id}"]`).should('exist');
+  });
+
+  it('sort alphabetically (descending first)', () => {
+    cy.login(MAKER_LOGIN);
+    // login and go to dashboard
+    gefDashboard.visit();
+
+    gefDashboard.tableHeader('exporter').find('button').click();
+    gefDashboard.tableHeader('exporter').find('button').click();
+    // makes sure is set to descending after clicking twice
+    gefDashboard.tableHeader('exporter').invoke('attr', 'aria-sort').should('eq', 'descending');
+
+    cy.get('table tr').eq(4).find(`[data-cy="deal__exporter--${deals[3]._id}"]`).should('exist');
+    cy.get('table tr').eq(3).find(`[data-cy="deal__exporter--${deals[2]._id}"]`).should('exist');
+    cy.get('table tr').eq(2).find(`[data-cy="deal__exporter--${deals[1]._id}"]`).should('exist');
+    cy.get('table tr').eq(1).find(`[data-cy="deal__exporter--${deals[0]._id}"]`).should('exist');
+  });
```

Comment:
Do we need to assert the exporter text or is "it exists" enough?

Reply:
@IgnatG ? I thought that that is enough as just making sure is right id there

Reply:
done